src/commit.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+echo "::group::Commit"
+
+cd "$INPUT_PACKAGE_NAME"
+
+echo "Push the new PKGBUILD and .SRCINFO files to the AUR repo"
+git add PKGBUILD .SRCINFO
+commit "$(generate_commit_message "" "$NEW_RELEASE")"
+git push
+
+if [[ "$INPUT_UPDATE_PKGBUILD" != "true" || -z "$INPUT_AUR_SUBMODULE_PATH" ]]; then
+    echo "Skipping submodule update and PKGBUILD update"
+    echo "::endgroup::Commit"
+    exit
```

Comment:
We are supposed to not exit the entire script if the condition is met

Reply:
You just have to skip the `:Commit::Main Repo` group and complete the rest of the script